github/ci/prow-deploy/vars/kubevirt-prow-workloads/main.yml

```diff
@@ -0,0 +1,17 @@
+testinfra_dir: /tmp/test-infra
+kubectl_exec: kubectl
+bootstrap_full_config: false
+reconcile_github_webhooks: false
+rescale_pushgateway_deployment: false
+project_infra_root: /home/prow/go/src/github.com/kubevirt/project-infra
+secrets_dir: '{{ project_infra_root }}/github/ci/prow-deploy/kustom/overlays/kubevirt-prow-workloads/secrets/'
+components_dir: '{{ project_infra_root }}/github/ci/prow-deploy/kustom/components'
+kubeconfig_path: '{{ secrets_dir }}/kubeconfig'
+
+shell_environment:
+  KUBECONFIG: "{{ kubeconfig_path }}"
+
+SRIOVNodes: []
```

Comment:
Not sure where that comes from, but this looks strange?

Reply:
Shouldn't it be similar to the old config [here](https://github.com/kubevirt/project-infra/blob/1e9e4be956c9974473b189be76075eb6e1fdbdc0/github/ci/prow-deploy/vars/workloads-production/main.yml#L14), so that the smoke_tests are executed?

Reply:
I have removed this variable as we no longer use that capacity label for controlling the SRIOV jobs - https://github.com/kubevirt/project-infra/pull/2664/commits/8f77dfdb79c367b0acdaf109945615eb1ae7953b